twitter.css

```diff
@@ -133,6 +134,32 @@ a:hover{
 }
 
 
+
+@media screen and (max-width: 1580px) {
```

Comment:
Made the Nav bar responsive.  It was challenging to get rid of the white space in between the nav and the main content.